### PR TITLE
Add senhasegura DSM in docs/stability-support.md

### PR DESCRIPTION
**Pull request by lfraga**
Add senhasegura DSM in docs/stability-support.md with alpha stability level

Related with https://github.com/external-secrets/external-secrets/pull/1075#issuecomment-1116071792

@knelasevero

### CODE REVIEW

File: docs/stability-support.md [hunk 1]
```diff
@@ -14,20 +14,21 @@ We are currently in beta and support **only the latest release** for the time be
...
+| [Oracle Vault](https://external-secrets.io/latest/provider-oracle-vault)                                   |   alpha   |                                                                                   [@KianTigger](https://github.com/KianTigger) [@EladGabay](https://github.com/EladGabay) |
+| [Akeyless](https://external-secrets.io/latest/provider-akeyless)                                           |   alpha   |                                                                                                                      [@renanaAkeyless](https://github.com/renanaAkeyless) |
+| [Generic Webhook](https://external-secrets.io/latest/provider-webhook)                                     |   alpha   |                                                                                                                                    [@willemm](https://github.com/willemm) |
+| [senhasegura DevOps Secrets Management (DSM)](https://external-secrets.io/latest/provider-senhasegura-dsm) |   alpha   |                                                                                                                                      [@lfraga](https://github.com/lfraga) |
```
Comment (knelasevero) [inline docs/stability-support.md:31]: Is senhasegura explicitly not capitalized for brand purposes? If not, I would Capitalize the first letter here :)

Comment (knelasevero) [reply docs/stability-support.md:31]: Not mandatory, btw

Comment (lfraga) [reply docs/stability-support.md:31]: Yes, the brand is uncapitalized

**Pull request merged**